project/plugins.sbt

```diff
@@ -4,15 +4,15 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
 
 resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.8")
```

Comment:
In the PR-Bot it says 2.8.18 which obviously makes no sense but thought I'd mention

Reply:
yeah for some reason when I updated everything and pushed it then changed and told me to update everything further